src/config.ts

```diff
@@ -112,7 +112,9 @@ export function getKmsKeyId(): string {
 }
 
 export function getOrchToAuthSigningPublicKey(): string {
-  return process.env.ORCH_TO_AUTH_SIGNING_KEY;
+  const publicKey = process.env.ORCH_TO_AUTH_SIGNING_KEY;
```

Comment:
this is a production change, need to keep an eye in the pipeline.

Reply:
100% agree. I added unit tests to ensure that the behaviour is the same for both with `\n` and with actual newline characters (`0x0a`).

The alternative here is to define the certs as multiline variables in the `.env.x` files:

```bash
ORCH_TO_AUTH_SIGNING_KEY="-----BEGIN PUBLIC KEY-----
MFkwEwYHKoZIzj0CAQYIKoZIzj0DAQcDQgAENB3csRUIdoaTHNn079Jl7JpiXzxF
0p2ZIddCErxtIhGMTTqtbQZJCPesSKUVE/DQbpIko3mLoisuFgmQfFouCw==
-----END PUBLIC KEY-----"
```

This way the environment variable is explicitly configured the same way as it is in the running environments. However, it’s a multiline bash variable and that has the potential to trip people up, so defining with `\n` seems overall safer.

Thoughts?